openwisp_radius/api/serializers.py

```diff
@@ -266,6 +270,41 @@ class Meta:
         read_only_fields = ('organization',)
 
 
+class UserGroupCheckSerializer(serializers.ModelSerializer):
+    result = serializers.SerializerMethodField()
+
+    class Meta:
+        model = RadiusGroupCheck
+        fields = ('attribute', 'op', 'value', 'result')
+
+    def get_result(self, obj):
+        counter = app_settings.CHECK_ATTRIBUTE_COUNTERS_MAP.get(obj.attribute)
```

Comment:
I think this could fail, can you double check if other parts of the code are handling this possible failure?